tests/test_model.py

```diff
@@ -7,6 +7,6 @@ def test_that_github_model_is_parsed_correctly():
     """
     github_json = {"pr": {"title": "Tie-tle"}}
 
-    dsl = GithubDSL.from_dict(github_json)
+    dsl = GitHubDSL(**github_json)
```

Comment:
What does this do?

Reply:
This is a leftover test code when the `GitHubDSL` was hand-coded by me, so I need to remove it anyway, that's a great place to comment 😄 

The `**` converts the dictionary to function method arguments, so:

```python
def function_with_parameters(first: str, second: int):
    pass

function_with_parameters(first="a", second=2)
```

is equivalent to:

```python
kwargs = {
    "first": "a",
    "second": 2
}
function_with_parameters(**kwargs)
```

You can also make your function take in any named arguments like this:

```python
def function_with_any_parameters(**kwargs):
    pass

function_with_any_parameters(param1="A", param2="B") 

# kwargs == {"param1": "A", "param2": "B"}
```

The same goes for non-keyworded arguments:

```python
def function_with_multiple_parameters(*args):
    pass

function_with_multiple_parameters(1, 2, 3, 4)
# args == [1, 2, 3, 4]
```

Reply:
Also, this change happened because the [pydantic](https://pydantic-docs.helpmanual.io/) library uses `**kwargs` convention to reconstruct the object from the dictionary.